bcda/cclf/importer.go

```diff
@@ -33,9 +33,13 @@ type cclf8Importer struct {
 }
 
 // validates that cclf8Importer implements the interface
-var _ importer = &cclf8Importer{}
+// var _ importer = &cclf8Importer{}
```

Comment:
Can you uncomment this out?

Reply:
scratch that, didn't notice that the interface was removed. We can delete the line now.